projects/ngx-quill/src/lib/quill-editor.component.ts

```diff
@@ -615,6 +622,22 @@ export abstract class QuillEditorBase implements AfterViewInit, ControlValueAcce
 
     return valid ? null : err
   }
+
+  private debounce<T extends(...args: any[]) => any>(callback: T): ((...args: Parameters<T>) => void) {
+    let timer: number
+    return (...args: Parameters<T>) => {
+      if (typeof this.debounceTime !== 'number') {
+        callback(...args)
+        return
+      }
+
+      clearTimeout(timer)
```

Comment:
could you please use window.clearTimeout and setTimeout?

Reply:
Sure, I hope the `this.doc.defaultView` is the preferred way to access the `window` object.